lib/advisor/core/questions/questions.yml

```diff
@@ -1,21 +1,17 @@
 technical:
   - "How could this person become a better pair?"
-  - "How could the this person improve their code quality?"
+  - "how could the this person improve their code quality?"
   - "How could this person improve their testing strategy?"
   - "How are this persons code reviews?"
```

Comment:
I think the question on line 5 would read better as `How is this person at code reviews?`

Reply:
Same as above 😄 Thanks though!